mail/gnupg-stable/Portfile

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+set my_name         gnupg
```

Comment:
What's the deal with my_name?

Reply:
That's a common pattern used when the port name does not match the software name (for instance when capitalization differs.) `git grep my_name` will provide you with a lot of examples.

Normally, when you change the `name` variable, you also implicitly change other variables (think of `dist*` variables for instance.)

If the port name does not match the software name, you have to override settings that normally are inferred via `${name}`, thus losing the mentioned benefit.

By employing a helper variable like `my_name`, this behavior can be retained.